src/components/ContactsList/ContactHeaderRow.jsx

```diff
@@ -1,9 +1,45 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { withStyles } from '@material-ui/core/styles'
 
-const ContactHeaderRow = props => <div className="divider">{props.header}</div>
+import Paper from 'cozy-ui/transpiled/react/Paper'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+const customStyles = theme => ({
+  paper: {
+    position: 'sticky',
+    zIndex: 1,
+    top: 0,
+    backgroundColor: 'var(--defaultBackgroundColor)',
```

Comment:
Instead of the CSS variable, please access directly the theme since you are in a MUI context theme.background.default.

Reply:
🤦